runtime/vm/createramclass.cpp

```diff
@@ -2804,10 +2809,6 @@ internalCreateRAMClassFromROMClass(J9VMThread *vmThread, J9ClassLoader *classLoa
 					J9ROMClass *packageRomClass = romClass;
 					U_32 pkgNameLength = 0;
 
-					if (J9ROMCLASS_IS_ARRAY(romClass)) {
```

Comment:
Also not sure why this was deleted.

Reply:
We no longer need the array check here because array classes are simply assigned the same module as their element class.